lib/datasource/cdnjs/index.ts

```diff
@@ -56,7 +56,7 @@ export async function getPkgReleases({
     const { assets, homepage, repository } = cdnjsResp;
 
     const releases = assets
-      .filter(({ files }) => files.indexOf(assetName) !== -1)
+      .filter(({ files }) => files.includes(assetName))
```

Comment:
Is this part of the new rule, or did you just add it?

Reply:
Part of the new rule and done by the auto fixer